source/administration/incident-response-application.rst

```diff
@@ -0,0 +1,667 @@
+ - Click **Choose .json file** and select the JSON file containing the app workflow schema.
+ - Click **Upload**.
+
+5. Post the trigger word defined in the app workflow schema to start the workflow!
+
+See the documentation below to get help on how to configure and manage your app workflows, or open an issue at https://forum.mattermost.org and we'd be happy to help you.
+
+Incident Response App Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The app is configured through a JSON file, which you can upload as a System Administrator via ``/workflow edit``. We highly recommend using a text editor file that automatically validates syntax errors of your JSON file.
+
+The JSON file looks something as follows.
+
+  .. code-block:: json
+
+    [
+      {
+        "name": "security_issue",
+        "triggers": [
+          {
+            "name": "issuereported",
+            "type": "text",
+            "match": "#sec"
+          }
+        ],
+        "steps": [
+          {
+            "name": "Setup",
+            "start_actions": [
+              {
+                "name": "CreateWarroom",
+                "type": "create_channel",
+                "channel_name": "security-issue-{{.Instance.Number}}",
+                "channel_displayname": "Security Issue {{.Instance.Number}}"
+              },
+              ...
+              {
+                "type": "transition_to",
+                "name": "TransitionToTriage",
+                "to": "Triage"
+              }
+            ]
+          },
+          ...
+          {
+            "name": "Resolved",
+            "start_actions": [
+              {
+                "name": "ResolveWarroom",
+                "type": "post",
+                "channel_name": "{{.Action.CreateWarroom.ChannelName}}",
+                "message": "Resolved."
+              },
+              ...
+              {
+                "name": "finished",
+                "type": "finished_workflow"
+              }
+            ]
+          }
+        ]
+      }
+    ]
+
+There are four components to an incident app workflow schema: triggers, steps, actions and transitions. Each one is explained in more detail below.
+
+.. note::
+  All field names must be a sequence of one or more letters, digits and underscores, and start with a letter.
+  
+  For instance, a trigger name can be ``sec_issue_workflow_3`` or ``SecIssueWorkflow3`` but not ``sec-issue-workflow-3``.
+
+Trigger
+^^^^^^^^^^^^^^^
+
+A trigger is a condition on which a workflow can be started.
+
+.. csv-table::
+    :header: "Field", "Description", "Type", "Required"
+
+    "name", "The name of the trigger", "string", "Yes. This field must be unique."
+    "type", "The type of trigger. This refers to the internal name of the trigger being configured", "string", "Yes"
+    "*additional fields", "Additional fields based on the type of trigger. See below for more details.", "string", "Yes"
```

Comment:
```suggestion
    "additional fields", "Additional fields based on the type of trigger. See below for more details.", "string", "Yes"
```

Reply:
The star is to indicate there are additional fields but that the field name isn't `additional fields`. I'm definitely open to better ways of illustrating it

Reply:
Maybe add a space between the star and the text. (There will be a formatting issue with the current way)

Reply:
Good suggestion, will do that,